tests/chart_tests/test_astronomer_commander.py

```diff
@@ -41,7 +41,7 @@ def test_astronomer_commander_deployment_upgrade_timeout(self, kube_version):
         """
         docs = render_chart(
             kube_version=kube_version,
-            values={"astronomer": {"commander": {"upgradeTimeout": 600}}},
+            values={"astronomer": {"commander": {"upgradeTimeout": 997}}},
```

Comment:
Im surprised to see 997 here thats a curious number

Reply:
The prior value for the non-default test was 600, which is what the new default value is, so I set the non-default test value to something conspicuous.